database/dynamodb.go

```diff
@@ -29,34 +21,56 @@ type AWSCredentials struct {
 var db *dynamodb.DynamoDB
 
 func init() {
+	defer func() {
+		if err := recover(); err != nil {
+			fmt.Println(err)
+		}
+	}()
```

Comment:
should database functions have defer recovery block? 

We can keep a single recover block where the functions are being called rather than creating a recovery function for every new function.

Reply:
Sure will remove the Panics with error thrown to the user. @RitikJaiswal75

Reply:
Removed the Panics thrown to logging + error `Internal Server error` to the end user https://github.com/Real-Dev-Squad/feature-flag-backend/pull/32/commits/36f697e413868af73c75dd5a71ceaf5a66666f36#diff-8a93b4ae6fdbffe07fb8d1afb9805a3ca03417060e0c47368b0e6c2299f4984cR41

@RitikJaiswal75

Reply:
Resolving this comment @RitikJaiswal75.